6_temp_coop_fetch/out/coop_all_files.rds.ind

```diff
@@ -1,2 +1,3 @@
-hash: 59795c7efcc7526f0462cdac07698f21
+hash: 7566d6ac6bfe10e47a9a3a90065e827e
+freshened: 2021-12-10 05:44:22 -0600
```

Comment:
going to `scdel()` this and rename from .ind to .tind (time-indicator) to avoid confusion with the extra information

Reply:
Never mind, that isn't possible because the various `scipiper` functions check that it has a valid name for an indicator file.

Reply:
Modified this with an updated commit